pkgs/tools/graphics/pdf-sign/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, substituteAll
+, ghostscript
+, pdftk
+, poppler_utils
+, coreutils
+
+, python3
+, makeWrapper
+}:
+
+let
+  python-env = python3.withPackages (ps: with ps; [ tkinter ]);
+in
+stdenv.mkDerivation {
+  pname = "pdf-sign";
+  version = "unstable-2023-08-08";
+
+  src = fetchFromGitHub {
+    owner = "svenssonaxel";
+    repo = "pdf-sign";
+    rev = "98742c6b12ebe2ca3ba375c695f43b52fe38b362";
+    hash = "sha256-5GRk0T1iLqmvWI8zvZE3OWEHPS0/zN/Ie9brjZiFpqc=";
+  };
+
+  patches = [
+    (substituteAll {
```

Comment:
Can't we just add `ghostscript`, `pdftk`, `poppler_utils`, `coreutils` in the `buildInputs` so we can get rid of this patch?

Reply:
I don't know how that would work. I don't think just having dependencies listed in `buildInputs` would allow the python script to call them, as this is only a shebang script, no special builders were used.

Reply:
Have you tried ?

Reply:
Fair enough, let's keep it like that for the moment.